Client/Program.cs

```diff
@@ -1,54 +1,166 @@
-﻿using System;
-using System.Threading;
-using Hazelcast.Client;
-using Hazelcast.Config;
-
-
-/**
- *
- * This is boilerplate application that configures client to connect Hazelcast Cloud cluster.
- * After successful connection, it puts random entries into the map.
- *
- * See: https://docs.hazelcast.cloud/docs/csharp-client
- *
- */
- namespace Client
- {
-     class Program
-     {
-         public static void Main(string[] args)
-         {
-             Environment.SetEnvironmentVariable("hazelcast.client.cloud.url", "YOUR_DISCOVERY_URL");
-             Environment.SetEnvironmentVariable("hazelcast.client.statistics.enabled", "true");
-             var config = new ClientConfig();
-             config.GetGroupConfig()
-                 .SetName("YOUR_CLUSTER_NAME")
-                 .SetPassword("YOUR_CLUSTER_PASSWORD");
-             config.GetNetworkConfig().GetCloudConfig()
-                 .SetEnabled(true)
-                 .SetDiscoveryToken("YOUR_CLUSTER_DISCOVERY_TOKEN");
-
-             var client = HazelcastClient.NewHazelcastClient(config);
-             var map = client.GetMap<string, string>("map");
-             map.Put("key", "value");
-             if(map.Get("key").Equals("value"))
-             {
-                 Console.WriteLine("Connection Successful!");
-                 Console.WriteLine("Now, `map` will be filled with random entries.");
-             }
-             else {
-                 throw new Exception("Connection failed, check your configuration.");
-             }
-             var random = new Random();
-             while (true) {
-                 var randomKey = random.Next(100_000);
-                 map.Put("key" + randomKey, "value" + randomKey);
-                 map.Get("key" + random.Next(100_000));
-                 if(randomKey % 10 == 0 )
-                 {
-                     Console.WriteLine("map size: {0}", map.Size());
-                 }
-             }
-         }
-     }
- }
+﻿// Copyright (c) 2008-2021, Hazelcast, Inc. All Rights Reserved.
+// 
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// 
+// http://www.apache.org/licenses/LICENSE-2.0
+// 
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
```

Comment:
Can we remove this part?

Reply:
certainly - I aligned the project with our client project 'cos I wasn't aware of your practices, going to cleanup now